client/my-sites/sites/index.jsx

```diff
@@ -8,57 +8,38 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import createReactClass from 'create-react-class';
 import { connect } from 'react-redux';
-import page from 'page';
 import i18n from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import Card from 'components/card';
 import Main from 'components/main';
-import observe from 'lib/mixins/data-observe';
 import SiteSelector from 'components/site-selector';
-import { addSiteFragment } from 'lib/route';
-import getSites from 'state/selectors/get-sites';
-import { getSiteSlug } from 'state/sites/selectors';
-import { getSelectedSite } from 'state/ui/selectors';
 
 export const Sites = createReactClass( {
 	displayName: 'Sites',
 
-	mixins: [ observe( 'user' ) ],
-
 	propTypes: {
 		path: PropTypes.string.isRequired,
 	},
 
 	filterSites( site ) {
-		let path = this.props.path;
-
-		// Override the path to be /sites so that when a site is
-		// selected the filterbar operates as if we're on /sites
-		if ( this.props.selectedSite ) {
-			path = '/sites';
-		}
+		const path = this.props.path;
 
 		// Filter out jetpack sites when on particular routes
 		if ( /^\/customize/.test( path ) ) {
 			return ! site.jetpack;
 		}
 
 		// Filter out sites with no upgrades on particular routes
-		if ( /^\/domains/.test( path ) || /^\/plans/.test( this.props.sourcePath ) ) {
-			return ! site.isJetpack || site.isSiteUpgradable;
+		if ( /^\/domains/.test( path ) || /^\/plans/.test( this.props.basePath ) ) {
```

Comment:
`this.props.basePath` is no longer present on the component. Can be replaced with just `path`.